P3-The-ViewController/content.md

```diff
@@ -3,7 +3,7 @@ title: "The ViewController"
 slug: the-viewcontroller
 ---     
 
-View controllers are a vital link between an app's data and its visual appearance. Whenever an iOS app displays a user interface, the displayed content is managed by a view controller or a group of view controllers coordinating with each other. Therefore, view controllers provide the skeletal framework on which you build your apps.
+View controllers are a vital link between an app's data and its visual appearance. Whenever an iOS app displays a user interface, the displayed content is managed by a view controller or a group of view controllers coordinating with each other. View controllers are containers for views. Therefore, view controllers provide the skeletal framework on which you build your apps.
```

Comment:
Hmmm, I don't think I agree with this wording - are you referring specifically to View Controllers in Storyboard?